<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CustomerEditDialog</class>
 <widget class="QDialog" name="CustomerEditDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>加入</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" >
      <widget class="QLineEdit" name="lineEditEmploy" />
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>备注</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEditFullName" />
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="lineEditPhone" />
     </item>
     <item row="3" column="1" colspan="3" >
      <widget class="QLineEdit" name="lineEditCompany" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>编号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEditCustomerNo" />
     </item>
     <item row="2" column="2" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>职业</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3" >
      <widget class="QLineEdit" name="lineEditAddress" />
     </item>
     <item row="4" column="2" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>手机</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QComboBox" name="comboBoxGender" >
       <item>
        <property name="text" >
         <string>女</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>男</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>地址</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>生日</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QDateEdit" name="dateEditBirth" >
       <property name="displayFormat" >
        <string>yyyy年MM月dd日</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" >
      <widget class="QLineEdit" name="lineEditMobile" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>单位</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>电话</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QDateEdit" name="dateEditAddDate" >
       <property name="displayFormat" >
        <string>yyyy年MM月dd日</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3" >
      <widget class="QTextEdit" name="textEditRemark" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>lineEditFullName</tabstop>
  <tabstop>comboBoxGender</tabstop>
  <tabstop>lineEditCustomerNo</tabstop>
  <tabstop>dateEditAddDate</tabstop>
  <tabstop>dateEditBirth</tabstop>
  <tabstop>lineEditEmploy</tabstop>
  <tabstop>lineEditCompany</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>lineEditMobile</tabstop>
  <tabstop>lineEditAddress</tabstop>
  <tabstop>textEditRemark</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomerEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
